dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/entity/User.java

```diff
@@ -221,6 +221,20 @@ public void setState(int state) {
         this.state = state;
     }
 
+    /**
+     * Get user statistics information id.
+     *
+     * This id decide what statistics information the user can see, if user is admin,
+     * it could see all statistics information, so we return `null` to unlimited statistics
+     * scope, but if user is general user, it could only see himself statistics information
+     * filtering data by user id.
+     *
+     * @return user Integer
+     */
+    public Integer getUserStatId() {
+        return this.userType == UserType.ADMIN_USER ? null : this.id;
+    }
+
```

Comment:
For reviewers, I add attribute `getUserStatId` and add desc about it, to reduce deplicate code in service.